pkg/util/mount/mount_helper.go

```diff
@@ -44,6 +46,28 @@ func CleanupMountPoint(mountPath string, mounter Interface, extensiveMountPointC
 	return doCleanupMountPoint(mountPath, mounter, extensiveMountPointCheck, corruptedMnt)
 }
 
+// CleanupNfsMountPoint is an unmount routine that unmounts the given path awared
+// of the possibility of hang caused by lost connection with nfs server.
+func CleanupNfsMountPoint(mountPath string, mounter Interface, extensiveMountPointCheck bool) error {
+	pathExists, pathErr := PathExistsWithTimeout(mountPath, 1*time.Second)
```

Comment:
how to decide the timeout is 1 second? Is it a little bit short?

Reply:
PathExists uses os.Stat() to decide if the dir exists, it would timeout only when nfs is disconnected, otherwise it returns immediately, so I used a minimum timeout and not expect nfs will reconnect in such period.

Reply:
how about make it a constant instead of embedding the number inside the code?

Reply:
sure, will do